common/trace/src/trace.rs

```diff
@@ -452,7 +452,7 @@ pub fn is_selected(node: (&'static str, u64)) -> bool {
         match &SAMPLING_CONFIG {
             Some(Sampling(sampling)) => {
                 if let Some(sampling_rate) = sampling.get(node.0) {
-                    node.1 % sampling_rate.denominator < sampling_rate.nominator
+                    node.1.wrapping_rem(sampling_rate.denominator) < sampling_rate.nominator
```

Comment:
I'm not sure I understand the difference between `%` and `wrapping_rem`? It also looks like none of them handle the case where the modulus is `0`: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b858206970d67a7f8d5299a533cd70d6

so weirdly this doesn't seem like it's good enough oO

Reply:
@changyonghu can the sampling_rate.denominator ever be 0 ?